src/main/java/com/ericsson/eiffel/remrem/semantics/SemanticsService.java

```diff
@@ -134,7 +134,8 @@ public SemanticsService(){
     }
     @Override
     public String generateMsg(String msgType, JsonObject bodyJson){
-        EiffelEventType eiffelType = EiffelEventType.fromString(msgType);
+        msgType = msgType.toLowerCase().replace("event", "")+"event";
```

Comment:
Add a comment for this line, Why we are doing this lowercase operation and "event" remove and concatination

Reply:
Format?